### PR TITLE
timetable docs

### PR DESCRIPTION
**Pull request by Faiz**
Docs for timetable API
docs

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -540,6 +540,246 @@ Error | Description
...
+# Timetable
+
+The base url is `https://uclapi.com/timetable/`
+
+Version Header | Latest Version
+-------------- | --------------
+`uclapi-search-version` | `1`
+
+## Get personal timetable
+**Endpoint:** `https://uclapi.com/timetable/get_timetable`
```
Comment (ChristopherHammond13) [inline source/index.html.md]: isn't this `/timetable/personal` ?

File: source/index.html.md [hunk 2]
```diff
@@ -540,6 +540,246 @@ Error | Description
...
+-d token=uclapi-5d58c3c4e6bf9c-c2910ad3b6e054-7ef60f44f1c14f-a05147bfd17fdb
+```
+
+```python
+import requests
+
+params = {
+  "token": "uclapi-5d58c3c4e6bf9c-c2910ad3b6e054-7ef60f44f1c14f-a05147bfd17fdb",
+}
+r = requests.get("https://uclapi.com/timetable/get", params=params)
```
Comment (ChristopherHammond13) [inline source/index.html.md]: `/get` doesn't seem to match either. Also I think this is OAuth only, no? Or can you use timetable with a standard token?

File: source/index.html.md [hunk 3]
```diff
@@ -540,6 +540,246 @@ Error | Description
...
+
+params = {
+  "token": "uclapi-5d58c3c4e6bf9c-c2910ad3b6e054-7ef60f44f1c14f-a05147bfd17fdb",
+}
+
+data = {
+    "modules": ["COMP3095"]
+}
+
+r = requests.post("https://uclapi.com/timetable/bymodule", params=params, data=data)
```
Comment (ChristopherHammond13) [inline source/index.html.md]: Again, this requires an OAuth token and therefore a `client_secret`

File: source/index.html.md (hunk 2 above)
Comment (Faiz) [reply source/index.html.md]: not with standar token. only oauth
I will update the urls.

File: source/index.html.md (hunk 3 above)
Comment (Faiz) [reply source/index.html.md]: aaahh client secret! I forgot that lol

File: source/index.html.md [hunk 4]
```diff
@@ -540,6 +540,249 @@ Error | Description
 `Token does not exist` | Gets returned when you supply an invalid `token`.
 `No query provided` | Gets returned when you have not supplied a `query` in your request.
 
+# Timetable
+
+The base url is `https://uclapi.com/timetable/`
+
+Version Header | Latest Version
+-------------- | --------------
+`uclapi-search-version` | `1`
```
Comment (wilhelmklopp) [inline source/index.html.md]: this says search - not timetable

File: source/index.html.md [hunk 5]
```diff
@@ -540,6 +540,249 @@ Error | Description
...
+
+## Get personal timetable
+**Endpoint:** `https://uclapi.com/timetable/personal`
+
+This endpoint returns personal timetable of the user.
```
Comment (wilhelmklopp) [inline source/index.html.md]: > This endpoint returns personal timetable of the user.

This endpoint returns the personal timetable of the user.

File: source/index.html.md [hunk 6]
```diff
@@ -540,6 +540,249 @@ Error | Description
...
+`sitename` | `Gordon House` | Name of the site where the event is happening.
+`type` | `CB` | Type of the room eg., Lecture Theatre, etc.
+`capacity` | `60` | Capacity of the room.
+`start_time` | `16:00:00` | Start time of the event.
+`end_time` | `18:00:00` | End time of the event.
+`module` | `JSON object` | Json object containing module details.
+`module_id` | `COMP3058` | Module id.
+`name` | `Artificial Intelligence and Neural Computing` | Module name.
+`course_owner` | `COMPS_ENG` | Department the module comes under.
+`module_code` | `COMP3C58` | Module code. We are not yet sure why this exists. Hopefully we will find out.
```
Comment (wilhelmklopp) [inline source/index.html.md]: could you delete module_code or module_id

File: source/index.html.md (hunk 5 above)
Comment (Faiz) [reply source/index.html.md]: the :D

File: source/index.html.md (hunk 6 above)
Comment (Faiz) [reply source/index.html.md]: done

**Pull request merged**